server/test/medium/specs/sync/sync-asset.spec.ts

```diff
@@ -23,12 +23,14 @@ describe.concurrent(SyncEntityType.AssetV1, () => {
   it('should detect and sync the first asset', async () => {
     const { auth, sut, getRepository, testSync } = await setup();
 
+    const originalFileName = 'firstAsset';
```

Comment:
Medium factory (asset insert) can optionally take one in the first argument object) and then return it so you can use asset.originalFilename in assertions I think